doc/sphinx/advanced_methods.rst

```diff
@@ -118,6 +118,63 @@ The following limitations currently apply for the collision detection:
 * The ``"bind at point of collision"`` approach cannot handle collisions
   between virtual sites
 
+.. _Deleting bonds when particles are pulled:
+
+Deleting bonds when particles are pulled
+----------------------------------------
+
+With this feature, bonds between particles can be deleted automatically
+when a critical bond extension is reached. This is useful to simulate
+weak bonds between particles.
+
+The bond breakage action is specified for individual bonds via the system
+:attr:`~espressomd.system.System.bond_breakage` attribute.
+
+Several modes are available:
+
+* ``"revert_center_bond"``: delete a bond from the first particle
```

Comment:
analog to the second item here: this can be used together with the collision detection: bond centers.